ui/raidboss/data/04-sb/trial/tsukuyomi-ex.ts

```diff
@@ -216,9 +237,13 @@ export default {
     },
     {
       id: 'Tsukuyomi Moonlit Debuff',
+      type: 'GainsEffect',
       netRegex: NetRegexes.gainsEffect({ effectId: '602' }),
-      condition: (data, matches) => matches.target === data.me && data.moonlitCount >= 4,
-      infoText: (_data, _matches, output) => output.text(),
+      condition: (data, matches) => {
+        const result = matches.target === data.me && data.moonlitCount && data.moonlitCount >= 4;
+        return !!result;
```

Comment:
Curious as to what the reasoning here is?
```suggestion
        return matches.target === data.me && data.moonlitCount && data.moonlitCount >= 4;
```

Reply:
Mmmm I think it's because `data.moonlitCount` is a number and so that makes this type `boolean | number`.  I'll see if I can make it better.